pyitt/pyproject.toml

```diff
@@ -2,7 +2,7 @@
 name = "pyitt"
 version = "1.1.0"
 authors = [
-  { name="Egor Suldin", email="rd3tap@yandex.ru" },
+  { name="Egor Suldin", email="egor.suldin@intel.com" },
```

Comment:
The usage of Intel email address is not align with Own-Time Open Source Project Approval that pyitt has. Therefore, please return the original email address back.

Reply:
If this is a script of "deployment" than it should be `ittapi` at the line number 2, the name of component owner and co-authors with their emails.

Reply:
I believe Egor is an Intel employee and copywright should be owned by Intel anyway

Reply:
For projects with Own-Time Open Source Project Approval, the copyright is held by their authors.